src/ForumFiltering.js

```diff
@@ -1,16 +1,18 @@
 import React from "react";
 import "./ForumFiltering.css";
+import axios from 'axios';
 
-function ForumFiltering (){
-    var keyword1 = {word: "Sports", type: "keyWords", checked: false}
-    var keyword2 = {word: "Community", type: "keyWords", checked: false}
-    var keyword3 = {word: "School", type: "keyWords", checked: false}
-    var keyword4 = {word: "Music", type: "keyWords", checked: false}
-    var keyword5 = {word: "Art", type: "keyWords", checked: false}
-    var cost1 = {word: "Free", type: "Cost", checked: false}
-    var cost2 = {word: "About 5 Dollars", type: "Cost", checked: false}
-    var cost3 = {word: "About 10 Dollars", type: "Cost", checked: false}
-    var cost4 = {word: "More than 10 Dollars", type: "Cost", checked: false}
+
+function ForumFiltering (props){
+    var keyword1 = {name: "keyword1", word: "Sports", type: "keyWords", checked: false}
+    var keyword2 = {name: "keyword2", word: "Community", type: "keyWords", checked: false}
+    var keyword3 = {name: "keyword3", word: "School", type: "keyWords", checked: false}
+    var keyword4 = {name: "keyword4", word: "Music", type: "keyWords", checked: false}
+    var keyword5 = {name: "keyword5", word: "Art", type: "keyWords", checked: false}
+    var cost1 = {name: "cost1", word: "Free", type: "Cost", checked: false}
+    var cost2 = {name: "cost2", word: "About 5 Dollars", type: "Cost", checked: false}
+    var cost3 = {name: "cost3", word: "About 10 Dollars", type: "Cost", checked: false}
+    var cost4 = {name: "cost4", word: "More than 10 Dollars", type: "Cost", checked: false}
 
```

Comment:
What is the "checked" attribute here?

Reply:
It's so later I can go in and check all the items and if they are "checked" then they will be fetched from the backend.